docs/sources/developers/plugins/migration-guide.md

```diff
@@ -33,6 +36,18 @@ This guide helps you identify the steps you need to take based on the Grafana ve
     - [Migrate to data frames](#migrate-to-data-frames)
   - [Troubleshoot plugin migration](#troubleshoot-plugin-migration)
 
+## From version 8.3.x to 8.4.x
+
+This section explains how to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x. Depending on your plugin, you need to perform one or more of the following steps.
+
+### Value Mapping Editor has been removed from @grafana-ui library
+
+Removed due to being an internal component.
+
+### Thresholds Editor has been removed from @grafana-ui library
+
+Removed due to being an internal component.
+
```

Comment:
Are these the only relevant lines?  the rest is just formatting updates?

We should be careful to not make this seem like a significant change.

Reply:
@mckn Specified to add a new section for this version (8.4)

Other changes are fixing broken links in current documentation (relrefs) / a few broken table of content links